README.md

```diff
@@ -17,6 +17,7 @@ ___
 [![Nightly-regression Test](https://github.com/openvinotoolkit/anomalib/actions/workflows/nightly.yml/badge.svg)](https://github.com/openvinotoolkit/anomalib/actions/workflows/nightly.yml)
 [![Pre-merge Checks](https://github.com/openvinotoolkit/anomalib/actions/workflows/pre_merge.yml/badge.svg)](https://github.com/openvinotoolkit/anomalib/actions/workflows/pre_merge.yml)
 [![Build Docs](https://github.com/openvinotoolkit/anomalib/actions/workflows/docs.yml/badge.svg)](https://github.com/openvinotoolkit/anomalib/actions/workflows/docs.yml)
+[![Downloads](https://static.pepy.tech/personalized-badge/anomalib?period=total&units=international_system&left_color=black&right_color=orange&left_text=PyPI%20Downloads)](https://pepy.tech/project/anomalib)
```

Comment:
Is it possible to change the color of the left part of the badge to grey (similar to the other badges)?